### DISCUSSION TITLE
Add an option that would allow you to get a report without the activities of the pg_profile itself

### DISCUSSION THREAD

**Discussion by Sshumeev** [zubkov-andrei/pg_profile#108] [Ideas]
Add an option that would allow you to get a report without the activities of the pg_profile itself

**Comment by Sshumeev** [comment]
UP

**Reply by zubkov-andrei** [reply]
Hi I think it is quite difficult to locate workload of pg_profle only. It's not clear to me what we could do with the cluster-wide statistics in that case. Anyway pg_profile footprint is quite small (and will be smaller in the) and if you see it in the report it means you do not have more heavy problems in your database.
